### DISCUSSION TITLE
Character animations not working.

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2467] [Q&A]
well with vs impostor i added the code with trial and error all by myself NO VIDEOS LAMO.
but the arrows in the character dont plays his animations even with Recoding NOTHING!
i am right now trying to see what is wrong but even while i do that EVEN WITH A VIDEO NOTHING! plus this is using kade engine
![image](https://user-images.githubusercontent.com/84847356/136475874-93a4d6db-a15e-4856-bf76-b646f737ae5a.png)
code:

playAnim('idle');

				setGraphicSize(Std.int(width * PlayState.daPixelZoom));
				updateHitbox();
				antialiasing = false;

				// DAD ANIMATION LOADING CODE
				case 'impostor':
				tex = Paths.getSparrowAtlas('impostor','shared',true);
				frames = tex;
				animation.addByPrefix('idle', 'impostor idle', 24, true);
				animation.addByPrefix('up', 'impostor up', 24, true);
				animation.addByPrefix('right', 'impostor right', 24, true);
				animation.addByPrefix('down', 'impostor down', 24, true);
				animation.addByPrefix('left', 'imposter left', 24, true);
imposter on left cause yes >:)
all he dose it FLOAT! 
![image](https://user-images.githubusercontent.com/84847356/136475944-c631c423-e219-468e-9e80-8bcac825f94a.png)

help me :(

**Comment by CarlitosPK** [comment]
The singing animations names must start with **sing**- pose

this new code should help you


```
animation.addByPrefix('idle', 'impostor idle', 24, true);
			animation.addByPrefix('singUP', 'impostor up', 24, true);
			animation.addByPrefix('singRIGHT', 'impostor right', 24, true);
			animation.addByPrefix('singDOWN', 'impostor down', 24, true);
			animation.addByPrefix('singLEFT', 'imposter left', 24, true);
```

Should help your problem

**Reply by glowsoony** [reply]
no i fixed it i was not using offsets that was the problem sense 1.6.2 add offsets they are meant to be used.
and btw i fixed that after 2 days
